playwright-e2e/tests/singular/enrollment/adult-dependent-enrollment-visual.spec.ts

```diff
@@ -15,11 +14,7 @@ async function signUp(page: Page) {
   await preScreeningPage.enterInformationAboutYourself();
 
   // Enter email alias and new password in Login popup
-  await fillEmailPassword(page, {
-    email: makeEmailAlias(process.env.singularUserEmail as string),
-    password: process.env.singularUserPassword,
-    waitForNavigation: true
-  });
+  await auth.createAccountWithEmailAlias(page);
```

Comment:
`createAccountWithEmailAlias` makes sense instead `fillEmailPasword` because a new account is to be created.